setup.py

```diff
@@ -4,11 +4,14 @@
 from setuptools import setup, find_packages
 from os         import path
 
+import cvmfs
+
+
 readme_path = path.join(path.dirname(__file__), 'README')
 
 setup(
-  name="python-cvmfsutils",
-  version="0.2.0",
+  name=cvmfs.package_name,
+  version=cvmfs.version,
```

Comment:
Hmmmm yes and no. Yes in the sense that it technically works. No in the sense that then you'd either have duplicity or you wouldn't include that data inside the actual package.
If you just want to the package name and revision inside setup.py then it's better to hardcode it, but creating a file in the same directory with this data means it won't be accessible from the cvmfs package.
In other words, you wouldn't be able to execute
````python
from cvmfs import cvmfs.__version__
```
because cvmfs wouldn't have a field named \_\_version\_\_

I don't know if I'm explaining...